src/main/java/seedu/address/model/person/Id.java

```diff
@@ -9,7 +9,10 @@
  */
 public class Id {
     public static final String MESSAGE_CONSTRAINTS =
-            "ID should only contain numbers and it should not be blank";
+            "ID should only contain positive numbers and it should not be blank. "
+                    + "Check the ID entered";
+    public static final String MESSAGE_ID_TOO_LARGE = "The ID entered is too large. "
+            + "Please verify the ID and try again.";
```

Comment:
This may be considered an enhancement and may not be allowed given the feature freeze. Changes to the error messages can only be more generic / general, and cannot give any more specific information about the type of error encountered.